containers/hugo/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM golang:1.13-alpine
+
+# VARIANT can be either 'hugo' for the standard version or 'hugo_extended' for the extended version.
+ARG VARIANT=hugo_extended
+# VERSION can be either 'latest' or a specific version number
+ARG VERSION=latest
+
+RUN apk add --update --no-cache ca-certificates openssl git curl && \
+    case ${VERSION} in \
+    latest) \
+    export VERSION=$(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4)}') ;;\
+    esac && \
+    echo ${VERSION} && \
+    wget -O ${VERSION}.tar.gz https://github.com/gohugoio/hugo/releases/download/v${VERSION}/${VARIANT}_${VERSION}_Linux-64bit.tar.gz && \
+    tar xf ${VERSION}.tar.gz && \
+    mv hugo* /usr/bin/hugo && \
+    go get github.com/yaegashi/muslstack && \
+    muslstack -s 0x800000 /usr/bin/hugo
+
+FROM alpine:edge
```

Comment:
Just as a FYI, there's also a base image you could use here that has things like git, even zsh pre-installed.  Just reference `mcr.microsoft.com/vscode/devcontainers/base:alpine`. It's built out of the `alpine` container folder in the vscode-dev-containers repo and mainly runs [this script](https://github.com/microsoft/vscode-dev-containers/blob/master/script-library/common-alpine.sh). 

Not a requirement to use it, just a convenience if it applies.

Reply:
My reference for the Hugo setup said that alpine:edge was required, but after testing the base image appears to work.